Generate_tile.scad

```diff
@@ -24,8 +24,11 @@ $fn=50;
 font = "Liberation Sans";
 
 letter_size = 4;
-letter_height = 5;
+letter_height = 2;
 
+// Round corners alter dimensions
+final_width  = (tile_width - (rounding_factor * 2));
+final_length = (tile_length - (rounding_factor * 2));
```

Comment:
The rounding_factor was affecting the tile dimensions with and length so I created final_x on lines 30 and 31 to subtract the rounding_factor deviation and keep the intended dimensions. Then I used final_width and final_length in the rest of the script